CoEpi/storage/CENKey.swift

```diff
@@ -0,0 +1,89 @@
+import Foundation
+import CryptoKit
+import RealmSwift
+import Security
+
+struct CENKey : Codable {
+    var timestamp: Int = Int(Date().timeIntervalSince1970)
+    var cenKey: String?
+    
+    //static var cenKey: String = ""
+    static var cenKeyTimestamp: Int = 0
+    
+    static func generateAndStoreCENKey() -> CENKey {
```

Comment:
Should these be static? Feels like there should be a `CENKeyRepo` object that's an injected singleton at the Dip layer, and it can govern these CENKey CRUD operations